### ISSUE TITLE
animated pixel shaders not working

### ISSUE THREAD

**Issue by ktownsend-personal** [microsoft/terminal#9058]
# Environment

```none
Windows build number: 10.0.18363.1316
Windows Terminal version (if applicable): preview 1.6.10272.0

Any other software? none relevant to issue
```

# Steps to reproduce

Configure an animated pixel shader.

# Expected behavior

The animated pixel shader should work.

# Actual behavior

Initial state of the shader appears, but then nothing changes. No error dialog. 

Non-animated shaders are working as expected. Just the animated shaders are failing to animate. I have tried both example animated shaders: Animate_scan.hlsl and Animate_breathe.hlsl.

**Comment by DHowett** [comment]
Those shaders were checked in as part of the fix for #8935. /dup #8935 :smile:

**Comment by ktownsend-personal** [comment]
thanks...I didn't spot the other issue when I did a search
